### ISSUE TITLE
Date format broke

### ISSUE THREAD

**Issue by alexherbo2** [Alexays/Waybar#569]
I have the following clock format, which renders `%-I` literally, instead of displaying the hour (01..12) without leading zeros.

``` json
{
  "clock": {
    "format": "{:%-I:%M %p}",
    "format-alt": "{:%A, %B %-d, %Y · %-I:%M %p}"
  }
}
```

**Comment by vired** [comment]
@alexherbo2 it is done automatically by `%I`. see https://howardhinnant.github.io/date/date.html#to_stream_formatting

**Comment by Alexays** [comment]
Fixed with #715 unless timezone is specified.
While waiting for a [HowardHinnant/date](https://github.com/HowardHinnant/date) update.
